app/src/main/assets/chart.html

```diff
@@ -48,7 +48,7 @@
       </tr>
       {% for row in rows %}
         {% set id = row.item.conceptIds | first %}
-        <tr class="obs concept-{{id}}">
+        <tr class="obs concept-{{id}} {{ row.item.label | tosafechars }}">
```

Comment:
Did you add the label as a CSS class because you tight it would be more friendly?  I can see why but I don't think it'll work well; the label will change for different languages so the best way to style the row is using the existing class with the  concept ID, which is stable.  We wouldn't want styles to break when someone adjusts the phrasing or spelling of a label.